monitoring/uptimerobot_create.sh

```diff
@@ -0,0 +1,19 @@
+# Latest distribution_config.txt can be found at
+# https://s3.console.aws.amazon.com/s3/object/harmony-benchmark/logs/2019/06/24/001727/distribution_config.txt?region=us-west-1&tab=overview
+# 
+
+create_monitors () {
+   cat distribution_config.txt | awk '{print $1,$4}' >  genesis_shard_ip.txt
```

Comment:
in the same aws folder, there is a hosts.txt contains all the ip address already.

Reply:
Lets use hosts.txt @john-harmony let me know if you need help getting it.